cpp/src/slice2swift/SwiftUtil.cpp

```diff
@@ -656,19 +648,14 @@ SwiftGenerator::validateMetadata(const UnitPtr& u)
 void
 SwiftGenerator::validateSwiftModuleMappings(const UnitPtr& unit)
 {
-    using ModuleMap = std::map<std::string, std::string>;
-    // Each Slice unit has to map all top-level modules to a single Swift module
-    ModuleMap mappedModules;
-    // With a given Swift module a Slice module has to map to a single prefix
-    std::map<std::string, ModuleMap> mappedModulePrefixes;
+    // Each Slice unit has to map all top-level modules to a single Swift module.
```

Comment:
It would be helpful to describe what this map is. It seems to be a map filename -> swift module name.

Reply:
I reworked this code to eliminate the need for the map.
The previous version was running this check even in included files, but this is overzealous.
Now we only run this check in the file that is actually being compiled.

And since we're only checking one file, no need for this map of filenames.

Reply:
The map was deleted.
The previous check was being applied to all files, even included ones.
This seemed over-zealous to me. Now we only check the single file that is being compiler.

And since we're only checking one file, we don't need a file-name map anymore.